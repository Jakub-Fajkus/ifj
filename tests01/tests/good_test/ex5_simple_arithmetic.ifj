class Main {
    static void run() {
        double d1 = 15.006;
        int i1 = 10;
	double tmp; //hodnota pretoze nemame rozsirenia	

        ifj16.print(i1);
	ifj16.print("\n");
        ifj16.print(d1);
	ifj16.print("\n");

	tmp = i1/d1;
        ifj16.print(tmp);
	ifj16.print("\n");

	tmp = i1*d1*110000*0.2218931;
        ifj16.print(tmp);

	ifj16.print("\n");
	tmp = d1-i1;
        ifj16.print(tmp);
	ifj16.print("\n");

        if(d1 != i1) 
	{
            ifj16.print(d1);
        } 
	else 

	{
	   ifj16.print(i1);
	}
	ifj16.print("\n");

        d1 = i1 * d1;
    }
}
