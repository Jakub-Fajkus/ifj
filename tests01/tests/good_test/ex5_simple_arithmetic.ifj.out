10
15.006
0.6664
3.6627e+06
5.006
15.006
